toolchain/win/BUILD.gn

```diff
@@ -203,10 +220,19 @@ template("msvc_toolchain") {
     tool("asm") {
       if (toolchain_args.current_cpu == "x64") {
         ml = "ml64.exe"
-      } else {
+      } else if (toolchain_args.current_cpu == "x86") {
         ml = "ml.exe"
       }
-      command = "$python_path $tool_wrapper_path asm-wrapper $env $ml {{defines}} {{include_dirs}} {{asmflags}} /c /Fo{{output}} {{source}}"
+      else if (toolchain_args.current_cpu == "arm"){
+        ml = "armasm.exe"
+      } else {
+        ml = "armasm64.exe"
```

Comment:
I think it needs to be an explicit check for arm64 in case another CPU is ever added.

Reply:
maybe assert "Unsupported CPU tartget"

Reply:
Updated